src/main/java/com/fasterxml/jackson/databind/deser/std/CollectionDeserializer.java

```diff
@@ -481,6 +487,22 @@ protected Collection<Object> _deserializeWithObjectId(JsonParser p, Deserializat
         return result;
     }
 
+    /**
+     * Deserialize the content of the collection.
+     * If _valueTypeDeserializer is null, use _valueDeserializer.deserialize; if non-null,
+     * use _valueDeserializer.deserializeWithType to deserialize value.
+     * This method only performs deserialization and does not consider _skipNullValues, _nullProvider, etc.
+     * @since 2.19.1
+     */
+    protected Object _deserializeRawContentValue(
+            JsonParser p,
+            DeserializationContext ctxt
+    ) throws IOException {
+        return _valueTypeDeserializer == null
+                ? _valueDeserializer.deserialize(p, ctxt)
+                : _valueDeserializer.deserializeWithType(p, ctxt, _valueTypeDeserializer);
```

Comment:
`_valueTypeDeserializer` and `_valueDeserializer` are `final` properties and were also being warned by `Intellij`, so they have been changed to refer to them directly.